templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -727,33 +727,65 @@ Resources:
           PolicyDocument:
             Version: 2012-10-17
             Statement:
-              - Effect: Allow
-                Action:
-                  - 'autoscaling:DescribeTags'
-                  - 'autoscaling:CreateOrUpdateTags'
-                  - 'ec2:AttachVolume'
-                  - 'ec2:CopySnapshot'
-                  - 'ec2:CreateSnapshot'
-                  - 'ec2:CreateTags'
-                  - 'ec2:CreateVolume'
-                  - 'ec2:DeleteSnapshot'
-                  - 'ec2:DescribeInstances'
-                  - 'ec2:DescribeSnapshots'
-                  - 'ec2:DescribeTags'
-                  - 'ec2:DescribeVolumes'
-                  - 'ec2:DetachVolume'
-                  - 'rds:AddTagsToResource'
-                  - 'rds:CopyDBSnapshot'
-                  - 'rds:CreateDBSnapshot'
-                  - 'rds:DeleteDBSnapshot'
-                  - 'rds:DescribeDBInstances'
-                  - 'rds:DescribeDBSnapshots'
-                  - 'rds:PromoteReadReplica'
-                  - 'rds:ModifyDBInstance'
-                  - 'rds:RestoreDBInstanceFromDBSnapshot'
-                  - 'iam:PassRole'
-                  - 'es:*'
-                Resource: ['*']
+              - Action:
+                  - ec2:DetachVolume
+                  - ec2:AttachVolume
+                  - ec2:CreateTags
+                  - ec2:CreateSnapshot
+                  - ec2:CreateVolume
+                Resource:
+                  - !Sub "arn:${AWS::Partition}:ec2:*:${AWS::AccountId}:instance/*"
+                  - !Sub "arn:${AWS::Partition}:ec2:*:${AWS::AccountId}:volume/*"
+                Effect: Allow
+              - Action:
```

Comment:
Can we group the permissions by service?

Reply:
Done